src/Components/Facility/FacilityConfigure.tsx

```diff
@@ -155,13 +155,15 @@ export const FacilityConfigure = (props: IProps) => {
               </div>
             </div>
             <div className="flex flex-col gap-3 sm:flex-row sm:justify-end">
-              <Cancel onClick={() => navigate(`/facility/${facilityId}`)} />
               <Submit onClick={handleSubmit} label="Update" />
             </div>
           </form>
         </div>
 
         <ConfigureHealthFacility facilityId={facilityId} />
+        <div className="mt-8 flex w-full justify-end">
+          <Back onClick={() => navigate(`/facility/${facilityId}`)} />
+        </div>
```

Comment:
This kinda looks odd. The back button that's present adjacent to the page title would be enough.

```suggestion
```